core/services/functions/listener.go

```diff
@@ -488,3 +508,18 @@ func (l *FunctionsListener) timeoutRequests() {
 		}
 	}
 }
+
+func (l *FunctionsListener) reportSourceCodeDomains(requestId [32]byte, domains []string) {
```

Comment:
It's not possible to do that in the adapter, correct?

Reply:
Technically it is possible, but this work will be significant. In core node we already have everything set for telemetry, hence the choice.

Reply:
Also I think we will be adding more telemetry data in future, and to me it is better to have a single telemetry emitter.